platform/ios/Mapbox-iOS-SDK-nightly-dynamic.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |m|
 
-  version = '3.6.2'
+  version = '3.6.3'
```

Comment:
I don't think we need / should update the nightly on the release branch. cc @friedbunny

Reply:
It’s not strictly necessary (as nightlies are built from `master`), but it doesn’t hurt to keep it in sync with its fellow podspecs, either.